noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -1219,40 +1661,15 @@
                 ]]>
             </errors>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
-        <title>Records within classes</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT r._ref_arkivdel series_system_id, r._ref_klasse class_system_id, COUNT(*) number_of_files
-                    FROM arkivstruktur.registrering r
-                    WHERE r._ref_mappe IS NULL
-                    GROUP BY r._ref_arkivdel, r._ref_klasse;
-                ]]>
-            </info>
-        </queries>
-    </rule>
-
-    <rule>
-        <title>Records without document descriptions</title>
-        <group>arkivstruktur</group>
-        <queries>
-            <info>
-                <![CDATA[
-                    SELECT r._ref_arkivdel series_system_id, COUNT(*) records_without_documents
-                    FROM arkivstruktur.registrering r
-                    WHERE r.systemid NOT IN (SELECT _ref_registrering FROM arkivstruktur.dokumentbeskrivelse WHERE _ref_registrering IS NOT NULL GROUP BY _ref_registrering)
-                    GROUP BY r._ref_arkivdel;
-                ]]>
-            </info>
-        </queries>
-    </rule>
-
-    <rule>
+    <test id="AST13">
         <title>Record status</title>
+        <description>
+            Tests whether a non-finalized Record exists in the archive.
```

Comment:
`Tests whether non-finalized records exist in the archive. A record is considered non-finalized if a finalized date or a finalizing party is not specified, or if the record status is not 'Arkivert' or 'Utgår'`